src/controllers/event/add.ts

```diff
@@ -118,15 +148,41 @@ const createEvent: Interfaces.Controller.Async = async (req, res, next) => {
       module: {
         connect: { id: moduleId },
       },
-      organizers: {
-        connect: organizersUsernames,
-      },
-      managers: {
-        connect: managersUsernames,
-      },
     },
   });
 
+  if (organizers) {
+    try {
+      organizers?.map(async (id: any) => {
+        await prisma.eventOrganiser.create({
+          data: {
+            userId: id,
+            eventId: event.id,
+          },
+        });
+      });
+    } catch (e) {
+      await prisma.event.delete({ where: { id: event.id } });
+      return next(Errors.Event.unableToCreate);
+    }
+  }
+
+  if (managers) {
+    try {
+      managers?.map(async (id: any) => {
```

Comment:
Same changes as mentioned before for organizers

Reply:
Removed the ? operator and gave the type string to the id.